.github/workflows/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
+  lighthouse:
+    name: Lighthouse
+    runs-on: ubuntu-22.04
+    steps:
```

Comment:
- [ ] This pull request has no summary/description and a proper title. Kindly use a more descriptive title and provide a detailed summary of the thing you did in the project. 

![image](https://user-images.githubusercontent.com/84156714/234553719-c389e6c4-ec84-49d2-9613-c3952a78914f.png)

E.g of a descriptive
- Title: Hello Microverse Project
- Summary: 👇🏾 
   ```
        Hello Microverse
        -  Add error checking linters.yml file
        -  Add a descriptive readme using markdown syntax
        -  Add HTML file
        -  Add CSS styling
```